### PR TITLE
Update .NET Framework Reference Assemblies version to 1.0.0-alpha-5

### PR DESCRIPTION
**Pull request by NikolaMilosavljevic**
Update version of .NET Framework reference assemblies package, to latest.

### CODE REVIEW

File: eng/Versions.props [hunk 1]
```diff
@@ -27,7 +27,7 @@
   <!-- Get .NET Framework reference assemblies from NuGet packages -->
   <PropertyGroup>
     <UsingToolNetFrameworkReferenceAssemblies>true</UsingToolNetFrameworkReferenceAssemblies>
-    <MicrosoftNetFrameworkReferenceAssembliesVersion>1.0.0-alpha-004</MicrosoftNetFrameworkReferenceAssembliesVersion>
+    <MicrosoftNetFrameworkReferenceAssembliesVersion>1.0.0-alpha-5</MicrosoftNetFrameworkReferenceAssembliesVersion>
```
Comment (livarcocc) [inline eng/Versions.props]: Is this really latest? Why the change in versioning format?

Comment (NikolaMilosavljevic) [reply eng/Versions.props]: It is the latest version we produced so far. I've discussed versioning with @dsplaisted offline before producing the packages and this was deemed a non-issue. Could we run into any issues with this versioning format change?

Comment (dsplaisted) [reply eng/Versions.props]: Looking at https://semver.org/, I think the correct formatting would be `1.0.0-alpha.5` to make sure the packages are sorted correctly if we're not padding with zeroes.

We don't need to fix this before updating to use the new packages though.

**Review by dsplaisted — APPROVED** [review]
